<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="save_path" value="$(find team30)/maps/map"/>

  <node pkg="map_server" type="map_saver" name="map_saver" output="screen" args="-f $(arg save_path)" /> />

</launch>